.github/workflows/publish-alpha.yml

```diff
@@ -0,0 +1,58 @@
+name: Publish Alpha Version to NPM
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: write # to be able to publish a GitHub release
+  issues: write # to be able to comment on released issues
+  id-token: write # to enable use of OIDC for npm provenance
+
+jobs:
+  pre-publish:
+    if: github.ref == 'refs/heads/develop'
+    uses: ./.github/workflows/publish-dry-run.yml
+    secrets:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+  publish:
+    needs: pre-publish
+    runs-on: ubuntu-latest
+    environment: development
+    # restricts job to develop branch
+    if: github.ref == 'refs/heads/develop'
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: 'yarn'
+
+      - name: Install dependencies
+        run: yarn --immutable
+
+      - name: Build
+        run: yarn build:packages
```

Comment:
looks like you need this command - `build-packages`

Reply:
fixed